docs/changelog.md

```diff
@@ -2,7 +2,11 @@
 
 ## 1.2 (unreleased)
 
-* Allow full-line comments in transactions (requires beancount > 2.0.0).
+* Updates for beancount 2.1.0:
```

Comment:
how do you know the next release version of beancount? I don't see any here: https://pypi.org/project/beancount/#history . Have Martin B. announced the future release version number?

Reply:
When I made this change, it was a guess (based on semver), but it is going to be 2.1.0, see https://bitbucket.org/blais/beancount/commits/cf4dd845a5473cd6e2960707d64f66d56270a834

```__version__ = '2.1.0'```